generator/main.cpp

```diff
@@ -53,6 +55,23 @@ void displayHelp(GeneratorSet *generatorSet);
 #include <QDebug>
 int main(int argc, char *argv[])
 {
+    /* Ensure printf function calls are always line buffered to avoid curious
+     * interleaving of the stdout and stderr streams when the output is piped
+     * to a file.  Ignore errors here (i.e. ignore closed stdout).
+     *
+     * This causes the check in test with win64_msvc2017_64 to fail.  The
+     * generator crashes here.  The cause is not known, it may be that the
+     * MSVC requires size != 0 even with a NULL buffer (ANSI-C is clear that
+     * the size is the size of the array).
+     */
+#   ifndef Q_CC_MSVC
+#       define _bufsize 0
```

Comment:
In the future, we should probably use definitions not dependent on qmake (not now though, they are used elsewhere).

Reply:
I just copied the usage in the rest of the code.  Anyway, it's gone now.